aksel.nav.no/website/components/token-docs/TokensPage.tsx

```diff
@@ -0,0 +1,36 @@
+"use client";
+
+import { useSearchParams } from "next/navigation";
+import React from "react";
+import { VStack } from "@navikt/ds-react";
+import TokenCategory from "./TokenCategory";
+import { TOKEN_CATEGORIES } from "./config";
+import { searchTokens } from "./toolbar/SearchField.utils";
+import Toolbar from "./toolbar/Toolbar";
+
+const TokensPage = () => {
+  const searchParams = useSearchParams();
+  const filteredTokens = searchTokens(searchParams?.get("tokenQuery") || "");
+  const filteredCategories = TOKEN_CATEGORIES.filter(({ id }) =>
+    filteredTokens.some((token) => token.category === id),
+  );
+  return (
+    <VStack gap="10">
```

Comment:
10 is a "legacy" value
```suggestion
    <VStack gap="space-40">
```

Reply:
Was changed to space-8 to reflect Figma